src/components/CharacterButton.vue

```diff
@@ -363,6 +372,10 @@ export default defineComponent({
     }
   }
 
+  .selected-row {
+    order: -1;
+  }
+
   .selected-character-item,
   .opened-character-item {
     background-color: rgba(colors.$primary-rgb, 0.2);
```

Comment:
`.selected-row`、`.selected-character-item`と役割が被ってそうだなとちょっと思いました！
良い感じに統合しちゃうか、ちょっと難しそうだったら↓みたいな感じでコメント残しても良いかもと思いました。

```suggestion
  // 選択中のキャラを上にする
  .selected-row {
    order: -1;
  }

  .selected-character-item,
  .opened-character-item {
    background-color: rgba(colors.$primary-rgb, 0.2);
```

Reply:
いい感じの統合案を思いつかなかったので、ご提案のようにコメントを残すようにします。